spec/triggers/user_added_to_group_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'USER_ADDED_TO_GROUP' do
+  before do
+    DiscourseAutomation::Scriptable.add('welcome_to_group') do
+      version 1
+
+      script do
+        p 'Howdy!'
```

Comment:
Debugging?

Reply:
No this is need, I actually capture stdout to ensure script has been running.